git/git.go

```diff
@@ -5,33 +5,57 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+
+	"github.com/mendersoftware/integration-test-runner/logger"
 )
 
-type gitCmd struct {
-	Dir string
-	cmd *exec.Cmd
+var dryRunMode bool
+
+func init() {
+	dryRunMode = false
+}
+
+// SetDryRunMode sets the dry run mode
+func SetDryRunMode(value bool) {
+	dryRunMode = value
+}
+
+// Cmd is a git command
+type Cmd struct {
```

Comment:
Why did we change `gitCmd` to be exported as `Git`? I don't see it in use from outside of this package.

Reply:
The linter complained about it